### DISCUSSION TITLE
公众号金玮律师被删文章：争取公开3-17岁人群接种新冠疫苗批准文件，药监局以可能危害社会稳定不公开

### DISCUSSION THREAD

**Discussion by WeNeedHome** [WeNeedHome/SummaryOfLoanSuspension#439] [General]
![8f851830d6f3059ea01bac93d78beeb](https://user-images.githubusercontent.com/109166931/179137924-c1fc5828-248e-44d3-b705-c3f3ee8eb06e.jpg)

**Comment by WeNeedHome** [comment]
![d1ffac446adb54113ec416831e1c201](https://user-images.githubusercontent.com/109166931/179138532-c1a57be1-331c-4101-b5f6-d616fb60fff9.jpg)

**Comment by radiateboy** [comment]
楼主别发这些无关的内容。

**Comment by wgli900222** [comment]
发这种玩意就变味了

**Comment by weizhanzhan** [comment]
和这个有啥关系

**Comment by qq1012827513** [comment]
别发这些无关的内容

**Comment by baoang** [comment]
这也无关，那亦无关，最后什么都是无关紧要的。
这也变味，那亦变味，最后什么都必须是正能量的。

**Reply by kennard520** [reply]
坚持不打

**Reply by songning1992** [reply]
我也坚持没打，我也不让我媳妇儿打，可惜了，前段时间我拦都拦不住我媳妇儿，一解封就非要去打那看似只有debuff的疫苗，唉，不知道说啥好，有些人他就是信那些xxx的话，你有什么办法。

**Comment by C-Joey** [comment]
要不你新开一个repo
